c/src/lib/libbsp/arm/shared/stm32fxxxx/error/hal-error.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * error.c
+ *
+ *  Created on: Jul 28, 2015
+ *      Author: jay.doyle
+ */
+
+/*
+ * Copyright (c) 2015 Vecna Technologies, Inc.
+ *
+ * The license and distribution terms for this file may be
+ * found in the file LICENSE in this distribution or at
+ * http://www.rtems.com/license/LICENSE.
+ */
+
+void stm32f_error_handler(
+  void
+)
+{
+    while(1) {
+        ; // Stay here forever
```

Comment:
This is just a place holder for now.  We talked about it in the last firmware meeting (maybe it was in the part that you missed at the start).  For now, it's useful just so that we can pause the execution and see what we were doing, but will likely replace this with some code to capture of state of execution in some log and/or print it to emulator debug serial stream.